GG_clone_backend/app/controllers/api/v1/unverified_drivers_controller.rb

```diff
@@ -0,0 +1,11 @@
+class Api::V1::UnverifiedDriversController < ApplicationController
```

Comment:
On my opinion we can write this functionality in drivers_controller for example, but we don't have time, dont change